benchmarks/DeepLearning/Ops/ArithAddfOp/GoogleBenchmarkMain.cpp

```diff
@@ -0,0 +1,142 @@
+//===- GoogleBenchmarkMain.cpp---------------------------------------------===//
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements the benchmark for addf operation.
+//
+//===----------------------------------------------------------------------===//
+
+#include <benchmark/benchmark.h>
+#include <buddy/Core/Container.h>
+#include <iostream>
+#include <random>
+
+// Define target layout.
+#define INPUT_N 1
+#define INPUT_H 32
+#define INPUT_W 32
+#define INPUT_C 16
+#define OUTPUT_N 1
+#define OUTPUT_H 32
+#define OUTPUT_W 32
+#define OUTPUT_C 16
```

Comment:
You can appropriately expand the data range(same to other cases) to reduce the impact of noise on performance. In my opinion, let the number of floating-point operations be around 1e8 is reasonable.